tasks/main.yml

```diff
@@ -77,5 +77,20 @@
     name: "{{ item }}"
     state: started
     enabled: yes
+    use: "service"
   with_items: "{{ varnish_enabled_services | default([]) }}"
-  when: varnish_enabled_services
+  when: >
+    varnish_enabled_services and
+    (ansible_os_family != 'Debian' and ansible_distribution_release != "xenial")
+
+# See: https://github.com/ansible/ansible/issues/22303
+- name: Ensure Varnish services are started enabled on startup (Xenial specific)
+  service:
+    name: "{{ item }}"
+    state: started
+    enabled: yes
+    use: "service"
```

Comment:
Another question here—isn't this task the exact same as the task above? I don't see why we'd need to have two tasks that do the exact same thing, but one for Xenial, and another for all other versions...